test/users-pagination-test.ts

```diff
@@ -0,0 +1,157 @@
+import axios from "axios";
+import { expect } from "chai";
+import jwt from "jsonwebtoken";
+import { prisma } from "../src/client/client";
+
+const port = process.env.PORT ? +process.env.PORT : 30002;
+const JWT_SECRET = process.env.JWT_SECRET!;
+
+describe('GET /users - Paginated Users', async () => {
```

Comment:
```suggestion
describe('GET /users', async () => {
```

Pra ficar igual os outros enunciados.